api/mailer/mailer.js

```diff
@@ -53,4 +53,16 @@ router.post('/validateVerificationEmail', async (req, res) => {
     });
 });
 
+router.post('/getCalendarEvents', async (req, res) => {
+  let calendarID = req.query.calendarID;
+  const numOfEvents = req.query.numOfEvents;
+  if (typeof calendarID === 'undefined') {
```

Comment:
this can become one `if` block
```suggestion
  if (numOfEvents < 0) {
    res.sendStatus(BAD_REQUEST);
  }
```